scripts/aem.js

```diff
@@ -612,6 +612,21 @@ async function loadBlocks(main) {
   }
 }
 
+/**
+ * Returns the value specified in front of key or undefined if missing in the block
+ * <div>
+ *  <div>key</div>
+ *  <div>value</div>
+ * <div>
+ * @param {*} block from which value needs to be extracted
+ * @returns the value or undefined if not present
+ */
+const getBlockKeyValue = (block, key) => {
```

Comment:
this should be handled via `readblockconfig` method in aem.js.
If there is requirement something to be customer ,this can move to block utils.

Reply:
Thanks @anagarwa for pointing this out, using readblockconfig instead of my own method